airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_assets_post.py

```diff
@@ -0,0 +1,75 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think this test should be directly in the same file name matching with the assets.py. Thanks for including the tests!